pkg/apk/fs/rwosfs_test.go

```diff
@@ -44,6 +44,9 @@ func TestExistingDir(t *testing.T) {
 		{"a/b/c", false, 0o644, []byte("hello")},
 		{"foo/bar", true, 0o700, nil},
 		{"foo/bar/world", false, 0o600, []byte("world")},
+		// ReadFile() fails on the following and doesn't fall back to
+		// trying to adjust the permissions.
+		// {"foo/bar/shadow", false, 0o000, []byte("shadow")},
```

Comment:
Do we need this hunk? either world is the same test case, or having a second file make a difference and then should be uncommented?